### PR TITLE
Gabriel fixes

### PR DESCRIPTION
**Pull request by GMart818**
Added blueDress test and page objects

### CODE REVIEW

File: page_objects/home.page.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ class HomePage extends Page {
 	 */
 	get signInButton () { return browser.element('#header a.login') }
 	get orangeDress() { return browser.element('[title="Printed Dress"].replace-2x')}
+	get bestSelllers() { return browser.element('[title="Best sellers"]')}
```
Comment (DaRuski) [inline page_objects/home.page.js]: Do you even spell, bro?

File: page_objects/bestSellers.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Page from './page'
+
+class BestSellersPage extends Page {
+
+        get fadedTop(){ return browser.element('[title="Faded Short Sleeve T-shirts"]')}
+        get colorBlue(){ return browser.element('#color_2')}
+
+        locateBestSeller(){
```
Comment (DaRuski) [inline page_objects/bestSellers.js]: Please be more descriptive in the naming of this function

File: test/specs/blueDress.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+
+
+describe('automation practice', function() {
+    it("user can select an item, change it's color, and add it to shopping cart" , function () {
+        HomePage.open();
+        HomePage.bestSelllers.click();
+        BestSellersPage.locateBestSeller();
+        LocateBestSellers.addButton.click();
+        let seeTheElement = browser.element('[title="Proceed to checkout"]');
```
Comment (DaRuski) [inline test/specs/blueDress.js]: Try and reuse elements that you have defined before. Also, make sure you name things descriptively.

File: test/specs/blueDress.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        seeTheElement.waitForVisible();
+        var rightAmount = browser.element('h2').getText();
+        assert.equal(rightAmount, 'Product successfully added to your shopping cart');
```
Comment (DaRuski) [inline test/specs/blueDress.js]: This is not the expected result that was asked of you to automate.

Comment (GMart818) [reply test/specs/blueDress.js]: Kirill, you forgot to mention that I'm a poop-head and forgot to follow instructions about browser.element

**Pull request merged**